src/taskpane/outlook.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global $, document, Office */
+
+import { getGraphData } from "./../helpers/ssoauthhelper";
```

Comment:
Just a comment that I think the "helpers" folder would be better off being named "sso" if it just contains SSO related functionality. No change needed but it would be nice to avoid generic terms like "helpers", "util", as these typically become dumping grounds for lots of different things rather than being cleanly separated.

Reply:
Changed tabs to spaces